smrf/distribute/solar.py

```diff
@@ -293,7 +293,7 @@ def distribute(self, date_time, cloud_factor, illum_ang, cosz, azimuth,
             # calculate clear sky radiation
 
             # Not all the clean but it will work for now
-            val_beam, val_diffuse = self.calc_stoporad(
+            val_beam, val_diffuse, horizon_angles = self.calc_stoporad(
```

Comment:
Performance improvement - Store the horizon_angles (aka shading) after calculating those in the IR. Same ones are needed for the VIS below.